src/plays/age-calculator-by-react/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Age Calculator by react
+
+Calculate your age by react
+
+## Play Demographic
+
+- Language: js
+- Level: Beginner
+
+## Creator Information
+
+- User: mahabub-bd
+- Gihub Link: https://github.com/mahabub-bd
+- Blog: 
+- Video: 
+
+## Implementation Details
```

Comment:
Please add `Implementation Details, Consideration`, and `Resources`. If not, you can remove that section.